src/SharedMoudule/Components/Filter/Filter.module.css

```diff
@@ -0,0 +1,24 @@
+.searchInput {
+    border: #c3c4ca solid 2px !important;
```

Comment:
It is not a good practice to write `!important` every where, this will make updating the styles later difficult.